x-pack/plugins/encrypted_saved_objects/server/saved_objects/index.ts

```diff
@@ -16,6 +16,7 @@ import { EncryptedSavedObjectsClientWrapper } from './encrypted_saved_objects_cl
 interface SetupSavedObjectsParams {
   service: PublicMethodsOf<EncryptedSavedObjectsService>;
   savedObjects: CoreSetup['savedObjects'];
+  getStartServices: CoreSetup['getStartServices'];
```

Comment:
The function is still using APIs from the `setup` contract (`savedObjects.addClientWrapper`) in addition to the `createInternalRepository` that moved to `start`, so I had to pass both the savedObject setup contract and the `getStartServices` resolver. Not sure if I should replace both by the root `CoreSetup` object instead?